ilastik/applets/tracking/structured/structuredTrackingGui.py

```diff
@@ -67,37 +67,37 @@ def _loadUiFile(self):
         
         parameters = self.topLevelOperatorView.Parameters.value        
-        if 'maxDist' in parameters.keys():
+        if 'maxDist' in list(parameters.keys()):
             self._drawer.maxDistBox.setValue(parameters['maxDist'])
-        if 'maxObj' in parameters.keys():
+        if 'maxObj' in list(parameters.keys()):
             self._drawer.maxObjectsBox.setValue(parameters['maxObj'])
-        if 'divThreshold' in parameters.keys():
+        if 'divThreshold' in list(parameters.keys()):
             self._drawer.divThreshBox.setValue(parameters['divThreshold'])
-        if 'avgSize' in parameters.keys():
+        if 'avgSize' in list(parameters.keys()):
             self._drawer.avgSizeBox.setValue(parameters['avgSize'][0])
-        if 'withTracklets' in parameters.keys():
+        if 'withTracklets' in list(parameters.keys()):
             self._drawer.trackletsBox.setChecked(parameters['withTracklets'])
-        if 'sizeDependent' in parameters.keys():
+        if 'sizeDependent' in list(parameters.keys()):
             self._drawer.sizeDepBox.setChecked(parameters['sizeDependent'])
-        if 'divWeight' in parameters.keys():
+        if 'divWeight' in list(parameters.keys()):
             self._drawer.divWeightBox.setValue(parameters['divWeight'])
-        if 'transWeight' in parameters.keys():
+        if 'transWeight' in list(parameters.keys()):
             self._drawer.transWeightBox.setValue(parameters['transWeight'])
-        if 'withDivisions' in parameters.keys():
+        if 'withDivisions' in list(parameters.keys()):
             self._drawer.divisionsBox.setChecked(parameters['withDivisions'])
-        if 'withOpticalCorrection' in parameters.keys():
+        if 'withOpticalCorrection' in list(parameters.keys()):
             self._drawer.opticalBox.setChecked(parameters['withOpticalCorrection'])
-        if 'withClassifierPrior' in parameters.keys():
+        if 'withClassifierPrior' in list(parameters.keys()):
             self._drawer.classifierPriorBox.setChecked(parameters['withClassifierPrior'])
-        if 'withMergerResolution' in parameters.keys():
+        if 'withMergerResolution' in list(parameters.keys()):
             self._drawer.mergerResolutionBox.setChecked(parameters['withMergerResolution'])
-        if 'borderAwareWidth' in parameters.keys():
+        if 'borderAwareWidth' in list(parameters.keys()):
             self._drawer.bordWidthBox.setValue(parameters['borderAwareWidth'])
-        if 'cplex_timeout' in parameters.keys():
+        if 'cplex_timeout' in list(parameters.keys()):
             self._drawer.timeoutBox.setText(str(parameters['cplex_timeout']))
-        if 'appearanceCost' in parameters.keys():
+        if 'appearanceCost' in list(parameters.keys()):
             self._drawer.appearanceBox.setValue(parameters['appearanceCost'])
-        if 'disappearanceCost' in parameters.keys():
+        if 'disappearanceCost' in list(parameters.keys()):
```

Comment:
FWIW, I think there's no need for conversion to `list()` here -- `in` works with iterators, too.